api4/channel_test.go

```diff
@@ -1974,6 +1978,21 @@ func TestUpdateChannelMemberSchemeRoles(t *testing.T) {
 	_, r1 := SystemAdminClient.UpdateChannelMemberSchemeRoles(th.BasicChannel.Id, th.BasicUser.Id, s1)
 	CheckNoError(t, r1)
 
+	timeout := time.After(600 * time.Millisecond)
+	waiting := true
+	for waiting {
+		select {
+		case event := <-WebSocketClient.EventChannel:
+			if event.Event == model.WEBSOCKET_EVENT_CHANNEL_MEMBER_UPDATED {
+				require.Equal(t, model.WEBSOCKET_EVENT_CHANNEL_MEMBER_UPDATED, event.Event)
+				waiting = false
+			}
+		case <-timeout:
+			require.FailNow(t, "Should have received event channel member websocket event and not timedout")
```

Comment:
Nit: `require.FailNow` terminates all tests. Usually you want just `require.Fail`

```suggestion
			require.Fail(t, "Should have received event channel member websocket event and not timedout")
```